health-services/individual/src/main/java/org/egov/individual/service/IndividualService.java

```diff
@@ -226,8 +246,147 @@ public List<Individual> delete(IndividualBulkRequest request, boolean isBulk) {
             populateErrorDetails(request, errorDetailsMap, validIndividuals, exception, SET_INDIVIDUALS);
         }
 
-        handleErrors(errorDetailsMap, isBulk, VALIDATION_ERROR);
+        handleErrors(isBulk, errorDetailsMap);
 
         return validIndividuals;
     }
+
+    private static void handleErrors(boolean isBulk, Map<Individual, ErrorDetails> errorDetailsMap) {
+        if (!errorDetailsMap.isEmpty()) {
+            log.error("{} errors collected", errorDetailsMap.size());
+            if (isBulk) {
+                log.info("call tracer.handleErrors(), {}", errorDetailsMap.values());
```

Comment:
You need to send the error to tracer in case of bulk. Please check the existing code!

Reply:
Made the necessary change to use the existing method